source/tests/test.ts

```diff
@@ -426,6 +426,12 @@ test('is.symbol', t => {
 	testType(t, 'symbol');
 });
 
+test('is.numericString', t => {
+	t.true(m.numericString('5'));
+	t.true(m.numericString('-3.2'));
+	t.true(m.numericString('Infinity'));
```

Comment:
The following test case should not pass, but does:

```js
t.true(m.numericString(''));
```

```js
Number('')
// => 0
```

We need to guard against empty string.

Reply:
Why not use `testType` here as well, instead of just checking several specific use cases?

The tests wouldn't even break here if you replaces the implementation to `return true;`.